drivers/SmartThings/zigbee-power-meter/src/bituo/init.lua

```diff
@@ -0,0 +1,373 @@
+  local raw_value = value.value * multiplier/divisor -- Uint V
+  device:emit_component_event(device.profile.components["PhaseA"], capabilities.voltageMeasurement.voltage({value = raw_value, unit = "V"}))
+end
+
+local function voltage_phaseB_handler(driver, device, value, zb_rx)
+  local component = device.profile.components["PhaseB"]
+  if component ~= nil then
+    local multiplier = device:get_field("RMSVOLTAGE_MULTIPLIER") or 1
+    local divisor = device:get_field("RMSVOLTAGE_DIVISOR") or 100
+    local raw_value = value.value * multiplier/divisor -- Uint V
+    device:emit_component_event(component, capabilities.voltageMeasurement.voltage({value = raw_value, unit = "V"}))
+  end
+end
+
+local function voltage_phaseC_handler(driver, device, value, zb_rx)
+  local component = device.profile.components["PhaseC"]
+  if component ~= nil then
+    local multiplier = device:get_field("RMSVOLTAGE_MULTIPLIER") or 1
+    local divisor = device:get_field("RMSVOLTAGE_DIVISOR") or 100
+    local raw_value = value.value * multiplier/divisor -- Uint V
+    device:emit_component_event(component, capabilities.voltageMeasurement.voltage({value = raw_value, unit = "V"}))
+  end
+end
+
+local function current_phaseA_handler(driver, device, value, zb_rx)
+  local multiplier = device:get_field("RMSCURRENT_MULTIPLIER") or 1
+  local divisor = device:get_field("RMSCURRENT_DIVISOR") or 100
+  local raw_value = value.value * multiplier/divisor -- Uint A
+  device:emit_component_event(device.profile.components["PhaseA"], capabilities.currentMeasurement.current({value = raw_value, unit = "A"}))
+end
+
+local function current_phaseB_handler(driver, device, value, zb_rx)
+  local component = device.profile.components["PhaseB"]
+  if component ~= nil then
+    local multiplier = device:get_field("RMSCURRENT_MULTIPLIER") or 1
+    local divisor = device:get_field("RMSCURRENT_DIVISOR") or 100
+    local raw_value = value.value * multiplier/divisor -- Uint A
+    device:emit_component_event(component, capabilities.currentMeasurement.current({value = raw_value, unit = "A"}))
+  end
+end
+
+local function current_phaseC_handler(driver, device, value, zb_rx)
+  local component = device.profile.components["PhaseC"]
+  if component ~= nil then
+    local multiplier = device:get_field("RMSCURRENT_MULTIPLIER") or 1
+    local divisor = device:get_field("RMSCURRENT_DIVISOR") or 100
+    local raw_value = value.value * multiplier/divisor -- Uint A
+    device:emit_component_event(component, capabilities.currentMeasurement.current({value = raw_value, unit = "A"}))
+  end
+end
+
+local refresh = function(driver, device, cmd)
+  device:refresh()
+  device:send(SimpleMetering.attributes.CurrentSummationDelivered:read(device))
+  device:send(ElectricalMeasurement.attributes.ActivePower:read(device))
+  device:send(ElectricalMeasurement.attributes.RMSVoltage:read(device))
+  device:send(ElectricalMeasurement.attributes.RMSCurrent:read(device))
```

Comment:
If the attribute has been added as a monitored attribute, it should automatically be read by `refresh`.

Please write some tests to verify this.